specs/phase0/fork-choice.md

```diff
@@ -472,6 +473,16 @@ def get_slot_component_duration_ms(basis_points: uint64) -> uint64:
     return basis_points * SLOT_DURATION_MS // BASIS_POINTS
 ```
 
+#### `get_attestation_due_ms`
+
+```python
+def get_attestation_due_ms(epoch: Epoch) -> uint64:
+    """
+    Calculate the duration until the attestation is due in milliseconds.
+    """
```

Comment:
I'd suggest removing this docstring. Not necessary for such simple functions.

Reply:
Got it, will remove this docstring